ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/queue-monitor.j2

```diff
@@ -0,0 +1,48 @@
+{#
+ Copyright (c) 2023 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - queue-monitor #}
+{% if queue_monitor_length is arista.avd.defined or queue_monitor_streaming is arista.avd.defined %}
+
+## Queue Monitor
+{%     if queue_monitor_length is arista.avd.defined %}
+
+### Queue Monitor Length
+
+{%         set enabled = queue_monitor_length.enabled | arista.avd.default('-') %}
+{%         set log = queue_monitor_length.log | arista.avd.default('-') %}
+{%         set default_high = queue_monitor_length.default_thresholds.high | arista.avd.default('-') %}
+{%         set default_low = queue_monitor_length.default_thresholds.low | arista.avd.default('-') %}
+{%         set notifying = queue_monitor_length.notifying | arista.avd.default('-') %}
+{%         set tx_latency = queue_monitor_length.tx_latency | arista.avd.default('-') %}
+{%         set cpu_high = queue_monitor_length.cpu.thersholds.high | arista.avd.default('-') %}
+{%         set cpu_low = queue_monitor_length.cpu.thersholds.low | arista.avd.default('-') %}
+
+| Enabled | Logging Interval | Default Thresholds High | Default Thresholds Low | Notifying | tx-latency | CPU Thresholds High | CPU Thresholds Low |
+| ------- | ---------------- | :-----------------------: | :----------------------: | --------- | ---------- | :-------------------: | :------------------: |
+| {{ enabled }} | {{ log }} | {{ default_high }} | {{ default_low }} | {{ notifying }} | {{ tx_latency }} | {{ cpu_high }} | {{ cpu_low }} |
```

Comment:
Maybe let's group the thresholds in one column as @ClausHolbechArista suggested on the internal chat